security/pkg/nodeagent/sds/mockcredfetcher.go

```diff
@@ -0,0 +1,26 @@
+package sds
```

Comment:
We could get rid of this file as well, just move FirstPartyJwt into sdsservice_test.go. Remove NewMockCredFetcher and reuse plugin. CreateMockInteTestPlugin(token)